app/assets/javascripts/statistics.es6

```diff
@@ -0,0 +1,43 @@
+export class Statistics { 
+  constructor(collection_data, work_data) {
```

Comment:
I was thinking `#work` or `#collection`, but maybe passing the dom reference would be even better?